charts/prometheus-pingdom-exporter/templates/deployment.yaml

```
@@ -18,6 +18,7 @@ spec:
       labels:
         {{- include "prometheus-pingdom-exporter.selectorLabels" . | nindent 8 }}
     spec:
+      serviceAccountName: {{ template "prometheus-pingdom-exporter.serviceAccountName" . }}
```

should that not be?

```suggestion
      serviceAccountName: {{ template "prometheus-pingdom-exporter.serviceAccount.name" . }}
```

also would seem to be empty by default?

---

I don't this this is correct because of `helpers.tpl` - https://github.com/prometheus-community/helm-charts/blob/main/charts/prometheus-pingdom-exporter/templates/_helpers.tpl#L53C1-L62

I get this error if i change it to `serviceAccount.name`
```
Error: template: prometheus-pingdom-exporter/templates/serviceaccount.yaml:7:20: executing "prometheus-pingdom-exporter/templates/serviceaccount.yaml" at <{{template "prometheus-pingdom-exporter.serviceAccount.name" .}}>: template "prometheus-pingdom-exporter.serviceAccount.name" not defined
```

---

I tend to use `fullnameOverride` so all resources are named the same. Would you say that there should be a default name for the service account?